src/testdir/test_unlet_env.vim

```diff
@@ -0,0 +1,25 @@
+" Test for unlet $FOOBAR.
```

Comment:
New file needs to be listed in a number of places. Find some commits where Bram adds new test file, I do not know the details.

Reply:
`src/Makefile` and `src/testdir/Make_all.mak`, or `src/testdir/test_alot.vim` should be updated.